src/components/Microsoft.DotNet.UpgradeAssistant/Checks/VisualBasicRazorTemplateCheck.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
This class will want to be moved to the new VB extension - whichever one gets merged in first, though